var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -39,6 +39,8 @@ class Caliper(CMakePackage):
     url      = ""
 
     version('master', git='https://github.com/LLNL/Caliper.git')
+    version('1.7.0',  git='https://github.com/LLNL/Caliper.git', tag='v1.7.0')
+    # version 1.6.0 is broken b/c it downloads the wrong gotcha version
     version('1.6.0',  git='https://github.com/LLNL/Caliper.git', tag='v1.6.0')
```

Comment:
I wonder if we should just remove 1.6.0 from the package if it's broken.

Reply:
I would agree, but somehow it seems one can't uninstall a version that isn't listed.